worldedit-core/src/main/java/com/sk89q/worldedit/command/GeneralCommands.java

```diff
@@ -145,22 +213,48 @@ public void timeout(Actor actor, LocalSession session,
 
     @Command(
         name = "/fast",
-        desc = "Toggle fast mode side effects"
+        desc = "Toggle fast mode"
     )
     @CommandPermissions("worldedit.fast")
-    public void fast(Actor actor, LocalSession session,
-                    @Arg(desc = "The side effect", def = "")
-                        SideEffect sideEffect,
-                    @Arg(desc = "The new side effect state", def = "")
-                        SideEffect.State newState,
-                    @Switch(name = 'h', desc = "Show the info box")
-                        boolean showInfoBox) throws WorldEditException {
+    @Deprecated
+    void fast(Actor actor, LocalSession session,
+              @Arg(desc = "The new fast mode state", def = "")
+                  Boolean fastMode) {
+        boolean hasFastMode = session.hasFastMode();
+        if (fastMode != null && fastMode == hasFastMode) {
+            actor.printError(TranslatableComponent.of(fastMode ? "worldedit.fast.enabled.already" : "worldedit.fast.disabled.already"));
+            return;
+        }
+
+        if (hasFastMode) {
+            session.setFastMode(false);
+            actor.printInfo(TranslatableComponent.of("worldedit.fast.disabled"));
+        } else {
+            session.setFastMode(true);
+            actor.printInfo(TranslatableComponent.of("worldedit.fast.enabled"));
+        }
+    }
+
+    @Command(
```

Comment:
This needs to be implemented properly.
![](https://i.imgur.com/fDr49C2.png)
You currently cannot click the status "On" or "Off". It will will up the incorrect help page for fast.

Reply:
Resolved, but we might need to think about `//perf` in general at some point I guess